backend/clubs/models.py

```diff
@@ -698,6 +698,31 @@ def send_approval_email(self, request=None, change=False):
                 reply_to=settings.OSA_EMAILS + [settings.BRANDING_SITE_EMAIL],
             )
 
+    def get_latest_and_latest_approved_versions(self):
+        """
+        Returns the latest and latest approved versions of club key fields
+        [name, description, image] contents that trigger reapproval.
+        """
+        latest_approved_version = (
+            self.history.filter(approved=True).order_by("-history_date").first()
+        )
+        latest_version = self.history.order_by("-history_date").first()
+
+        if (
+            not latest_version
+            or not latest_approved_version
+            or (latest_approved_version == latest_version)
+        ):
+            return {}
```

Comment:
Can't we just check: "is the club currently not approved, and did there exist a previous version that was approved".